src/Campaigns/Blocks/CampaignCover/edit.tsx

```diff
@@ -47,15 +47,6 @@ export default function Edit({attributes, setAttributes, toggleSelection}: EditP
     return (
         <figure {...blockProps}>
             <CampaignSelector attributes={attributes} setAttributes={setAttributes}>
-                {hasResolved && !campaign?.image && (
-                    <Placeholder
-                        icon={<GalleryIcon />}
-                        label={__('Campaign Cover Image', 'give')}
-                        instructions={__('Upload a cover image for your campaign.', 'give')}
-                    />
-
-                )}
-
```

Comment:
@JoshuaHungDinh I don't think we should remove it because it removes the visual feedback for the user on the block editor. 

**Before:**

![image](https://github.com/user-attachments/assets/2507cf22-57a2-4715-9704-aeb83e783f58)

**After:**

![image](https://github.com/user-attachments/assets/353f0ab4-33a3-4616-a04a-6d1e1986e86f)